runners/core-java/src/main/java/org/apache/beam/sdk/util/SideInputHandler.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.util;
```

Comment:
I wanted to say that the use of this namespace in the runners-core module is just so that the move from SDK to runners-core is a separate change from the package rename. I think here it is best to use `org.apache.beam.runners.core` directly.

Eventually it should all be in this namespace, to avoid split packages. This helps, for example, with shading, and makes import enforcement rules work better, etc.